.readthedocs.yaml

```
@@ -10,19 +10,15 @@ build:
   os: ubuntu-22.04
   tools:
     python: "mambaforge-4.10"
+  jobs:
+    post_create_environment:
+      - pip install -e .[develop]
```

```suggestion
      - pip install . --no-deps
```
might be even better because then no dependencies will get installed for sure and all the requirements are already in environment.yml: https://github.com/ESMValGroup/ESMValTool/blob/ccab48b0030b3b2eaf297f4b595e3927799dd790/environment.yml#L73-L76

---

ah good call, man, well spotted :beer: